packages/kbn-react-hooks/src/use_boolean/use_boolean.ts

```diff
@@ -1,8 +1,9 @@
 /*
  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
  * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
```

Comment:
I think you should also update this file https://github.com/tonyghiani/kibana/blob/refactor/use-boolean-extraction/x-pack/plugins/observability_solution/infra/public/pages/metrics/inventory_view/components/waffle/node_square.tsx#L15, don't you?

Reply:
Good catch! I just fixed it 👍